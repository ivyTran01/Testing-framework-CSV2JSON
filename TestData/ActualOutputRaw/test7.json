[
{"name":"Ivy","age":"","ice_cream_lover":true},
{"name":"Tanzil","age":25,"ice_cream_lover":false},
{"name":"Danny","age":21,"ice_cream_lover":true}
]
