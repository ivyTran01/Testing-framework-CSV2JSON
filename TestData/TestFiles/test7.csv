Ivy  T.	22	true	5.2
Tanzil	""	false	5.9
Danny  Yu	21	true	5.11
Abdullah  Amjad		true	5.10
Jocelyn	22	true	5.2
Christ	23	false	5.10
