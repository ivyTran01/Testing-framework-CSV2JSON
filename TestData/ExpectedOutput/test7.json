[{"name":"Ivy","age":null,"ice_cream_lover":true},{"name":"Tanzil","age":25.0,"ice_cream_lover":false},{"name":"Danny","age":21.0,"ice_cream_lover":true}]